contents/handbook/engineering/feature-ownership.md

```diff
@@ -51,6 +51,7 @@ You can also view the list [directly in GitHub](https://github.com/PostHog/posth
 | Replay | [Team Replay][Team Replay]  |  <span class="lemon-tag gh-tag">feature/replay</span> |
 | Retention | [Team Product Analytics][Team Product Analytics]  |  <span class="lemon-tag gh-tag">feature/retention</span> |
 | Saved Insights | [Team Product Analytics][Team Product Analytics]  |  <span class="lemon-tag gh-tag">feature/saved-insights</span> |
+| Security | [@benjackwhite][@benjackwhite] is a good reference person but it is every team's job to consider and react to security issues |  <span class="lemon-tag gh-tag">feature/security</span> |
```

Comment:
```suggestion
| Security | [@benjackwhite][@benjackwhite] is a good reference person but it is every teams job to consider and react to security issues |  <span class="lemon-tag gh-tag">feature/security</span> |
```

Reply:
isn't that what @pauldambra  suggested fixing - I edited your above comment btw before committing it